src/core_plugins/kibana/public/visualize/editor/editor.html

```diff
@@ -67,12 +67,19 @@
   </div>
 
   <visualize
+    ng-if="!chrome.getVisible()"
     saved-obj="savedVis"
     ui-state="uiState"
     time-range="timeRange"
     editor-mode="chrome.getVisible()"
```

Comment:
Can be removed.

Reply:
remove the `ng-if`? we can't because the editor directive used it when sharing and embedding the viz.
If you remove it it will show two visualization one inside the visualization-editor and one outside.

Reply:
we only want to show this if chrome is not hidden (in embedded mode)